### ISSUE TITLE
Dataset, Test

### ISSUE THREAD

**Issue by MichaelD7911** [zcablii/SARDet_100K#27]
At the data set link, there is only Train set and Validation set.
https://liveuclac-my.sharepoint.com/:f:/g/personal/zcablii_ucl_ac_uk/EtzHpeCVMcFDmVKhU_Z2hqQBhmkU9ZWKgtsES6PmT9rLKw?e=bMIhcb

Is there also Test set as appears at paper (https://arxiv.org/pdf/2403.06534.pdf) ?

**Comment by zcablii** [comment]
refer to #2 . The test set will be made public on 2024-09-03.
